src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -79,6 +79,8 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
 
         self.ubuntu_pro_client_all_updates_cached = []
         self.ubuntu_pro_client_all_updates_versions_cached = []
+        
+        self.package_install_expected_avg_time_in_minutes = 1.5 # As per telemetry data, the average time to install package is around 81 seconds for apt.
```

Comment:
nit: using seconds would be better since the value is in decimal

Reply:
I wanted to double check before making this change. 

As we have all the parameters in minutes in the functions get_max_batch_size() and is_package_install_time_available(), if we use seconds instead of minutes for package_install_expected_avg_time_in_minutes , then we will have to convert it to minutes in the above-mentioned methods. 

Should I still make it to seconds?

Reply:
We generally use seconds throughout our code. My suggestion would be to change mins to secs in get_max_batch_size()

Reply:
Addressed it. Using seconds